tough/Cargo.toml

```diff
@@ -17,10 +18,13 @@ olpc-cjson = { version = "0.1.0", path = "../olpc-cjson" }
 pem = "0.8.1"
 reqwest = { version = "0.10.4", optional = true, default-features = false, features = ["blocking"] }
 ring = { version = "0.16.13", features = ["std"] }
+rusoto_core = { version = "0.44", optional = true, default-features = false }
+rusoto_kms = { version = "0.44", optional = true, default-features = false }
```

Comment:
The purpose of placing the KMS feature in its own crate is to avoid introducing these dependencies to `tough`.

```suggestion
```